### PR TITLE
[bugfix] Fix cases with MINPV without PINCH or MINPV with PINCH NOGAP

### PR DESCRIPTION
**Pull request by blattms**
For MINPV without PINCH OPM flow still silent assumed there was "PINCH GAP" specified when processing MINPV and created NNC where none should have been created.
For the case where only one cell was deactivated we also missed checking whether the thickness of this cell was below the threshold for "PINCH NOGAP" before creating an NCC over it.

I am sligtly nervous that I needed to change our test of FIVE_PINCH_NOGAP.DATA, where we no don't create an NNC anymore. Somebody should double check that with the other simulator. Maybe that test was wrong all the time.

This fixes the problem for PINCH10_NOPINCH in OPM/opm-tests#886

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this please

File: tests/test_minpvprocessor.cpp [hunk 1]
```diff
@@ -67,9 +67,9 @@ BOOST_AUTO_TEST_CASE(Pinch)
     pinch_no_gap = true;
     minpv_result = mp1.process(thickness, z_threshold, pv, minpvv, actnum, fill_removed_cells, z1.data(), pinch_no_gap);
     BOOST_CHECK_EQUAL_COLLECTIONS(z1.begin(), z1.end(), zcornAfter.begin(), zcornAfter.end());
-    BOOST_CHECK_EQUAL(minpv_result.nnc.size(), 1);
+    BOOST_CHECK_EQUAL(minpv_result.nnc.size(), 0); // No nnc because deactivated cell is to broad
```
Comment (akva2) [inline tests/test_minpvprocessor.cpp:70]: Too.

**Comment by blattms** [comment]
I'll mark this as WIP again until tested.

File: tests/test_minpvprocessor.cpp (hunk 1 above)
Comment (blattms) [reply tests/test_minpvprocessor.cpp]: Thanks. Still learning. A bit embarrassing that I have been doing this mistake all my live.

Comment (bska) [reply tests/test_minpvprocessor.cpp]: In the same vein, what is "broad" here?  Thick?  Wide?  Long?

Comment (blattms) [reply tests/test_minpvprocessor.cpp]: reworded for better clarity (hopefully)

**Comment by blattms** [comment]
Marking this as ready for review as it seems to fix the immediate problem.
Please be aware that there are more issues with MINPV/PINCH that we will work on in future PRs.

**Comment by blattms** [comment]
jenkins build this please

**Comment by blattms** [comment]
This is now tested and fixes an important issue,

File: tests/test_minpvprocessor.cpp [hunk 2]
```diff
@@ -67,9 +67,9 @@ BOOST_AUTO_TEST_CASE(Pinch)
     pinch_no_gap = true;
     minpv_result = mp1.process(thickness, z_threshold, pv, minpvv, actnum, fill_removed_cells, z1.data(), pinch_no_gap);
     BOOST_CHECK_EQUAL_COLLECTIONS(z1.begin(), z1.end(), zcornAfter.begin(), zcornAfter.end());
-    BOOST_CHECK_EQUAL(minpv_result.nnc.size(), 1);
+    BOOST_CHECK_EQUAL(minpv_result.nnc.size(), 0); // No nnc because deactivated cell thickness is to high
```
Comment (akva2) [inline tests/test_minpvprocessor.cpp:70]: too

**Review by akva2 — APPROVED** [review]
Apart from the deja-vu it looks fine to me.

File: tests/test_minpvprocessor.cpp (hunk 2 above)
Comment (blattms) [reply tests/test_minpvprocessor.cpp:70]: I should get my writing skills tested. second problem in the same line :rofl:

**Comment by blattms** [comment]
jenkins build this please

**Pull request merged**